wrapper/src/test/java/integration/host/TestEnvironmentProvider.java

```diff
@@ -186,6 +192,15 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
                                 || config.noIam
                                 ? null
                                 : TestEnvironmentFeatures.IAM,
+                            deployment == DatabaseEngineDeployment.DSQL
```

Comment:
It seems that the following features are always on for DSQL:
REQUIRES_IAM_AUTH
REQUIRES_TLS_SNI
SUPPORTS_FOREIGN_KEYS

What is the purpose of them?